integr8ly/installation
# Add launcher postgres backup

## Additional Information
https://issues.jboss.org/browse/INTLY-1309

Sets up a CronJob to backup the Launcher postgres.

## Verification Steps

1. Create a secret in default named `s3-credentials` that contains your aws credentials
1. Update the master URL in managed.template and set core_install=false if you've already setup integreatly
1. Set backup_schedule in manifest.yaml to '*/1 * * * *' to run each minute
1. Run: ansible-playbook -i inventories/managed.template playbooks/managed/install.yml
1. Wait a minute
1. In the launcher namespace
6.1 Ensure the `launcher-postgres-backup` Pod succeeds
6.1 Check S3 and ensure the archives have been pushed up.
6.1 Delete all CronJobs otherwise theykeep creating backups every minute
